.github/workflows/main.yaml

```diff
@@ -1,9 +1,9 @@
 name: Oryx-SDK-Automation
 
 on:
-  # runs on 12:55 (5:55AM PST) & 23:55 UTC
+  # runs at 12pm PST per .NET team recommendation
```

Comment:
This means that if an SDK is released after 12pm PST on any given day, then our automation won't catch it, correct? Did the .NET team guarantee that it will always be released before then / did they understand that if it doesn't detect it at noon then it won't detect it at all?

Reply:
Depends on how our logic handles previous day releases -- I remember having a conversation with @william-msft about how this was implemented, and we ultimately settled on running the function twice a day since it may have difficulty with previous day releases. Is this still the case @william-msft ?

Reply:
Keeping current logic for now (adding checking if version exists in SDK soon)